scripts/asset-setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+pushd $PWD
+cd ./launch
+yarn && yarn run ts-node index.ts 
```

Comment:
`ts-node index.ts` should be a script in `package.json` and we can do something like:

```
yarn --cwd launch start
```

yarn can be put into `makefile`'s `init` target:

```
.PHONY: init
init: submodules
	git config advice.ignoredHook false
	git config core.hooksPath .githooks
	rustup target add wasm32-unknown-unknown
       // yarn --cwd launch
```